nexus-config/src/nexus_config.rs

```diff
@@ -592,19 +592,28 @@ pub struct PhantomDiskConfig {
 #[serde_as]
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct BlueprintTasksConfig {
+    /// background planner chicken switch
+    #[serde(default)]
```

Comment:
I'd suggest dropping the `serde(default)` since I think you updated all the copies of this config here anyway.